cli/cluster/membership.go

```diff
@@ -24,24 +24,34 @@ func (membershipInstance *MembershipInstance) DecodeMsgpack(d *msgpack.Decoder)
 	return common.DecodeMsgpackStruct(d, membershipInstance)
 }
 
+func GetFirstInstanceName(instances map[string]string) (string, bool) {
+	for _, instanceName := range instances {
+		return instanceName, true
+	}
+	return "", false
+}
+
+type InstancesFilter func(*MembershipInstances, map[string]string)
+
 // GetMembershipInstances returns MembershipInstances for currently configured instances
 // First, it connects all instances to membership (probes all running instances by one
 // of them). Then, it gets all membership instances members.
-func GetMembershipInstances(instancesConf *InstancesConf, ctx *context.Ctx) (*MembershipInstances, error) {
-	runningInstancesNames := getRunningInstances(instancesConf, ctx)
-	if len(runningInstancesNames) == 0 {
+func GetMembershipInstances(instancesConf *InstancesConf, ctx *context.Ctx,
+	filters ...InstancesFilter) (*MembershipInstances, error) {
+	runningInstances := getRunningInstances(instancesConf, ctx)
+	instanceName, ok := GetFirstInstanceName(runningInstances)
+	if !ok {
```

Comment:
Perhaps we dont need to return 2 values from GetFirstInstanceName,
 why couldnt we just check: `if instanceName == ""{
...
}`